src/Models/HydrostaticFreeSurfaceModels/compute_w_from_continuity.jl

```diff
@@ -19,7 +24,7 @@ end
 @kernel function _compute_w_from_continuity!(U, grid)
     i, j = @index(Global, NTuple)
     # U.w[i, j, 1] = 0 is enforced via halo regions.
-    @unroll for k in 2:grid.Nz
+    @unroll for k in 2:grid.Nz+1
```

Comment:
🌊

Reply:
This should be Nz+1?

Reply:
vertically velocity is non-zero at `k=Nz+1` when there's a free surface